relay/relay.go

```diff
@@ -307,6 +334,10 @@ func (rs *Relay) GetHeader(ctx context.Context, m *structs.MetricGroup, uc struc
 			return nil, ErrInternal
 		}
 
+		if rs.config.Distributed {
+			go rs.streamCacheBlock(maxProfitBlock.Payload)
```

Comment:
Assuming this may have a lot of the same slot being emitted, shouldn't we create some sort of a short lived cache, that this slot+blockhash was already sent?

Reply:
Very good call, yes.